helm/hpcc/templates/eclagent.yaml

```diff
@@ -172,7 +172,7 @@ spec:
 {{-  if .useChildProcesses }}
 {{- include "hpcc.addResources" (dict "me" .resources) | indent 8 }}
 {{-  else }}
-{{- include "hpcc.addStubResources" ($commonCtx | merge (dict "instances" .maxActive)) | indent 8 }}
+{{- include "hpcc.addStubResources" ($commonCtx | merge (dict "stubResources" .stubResources)) | indent 8 }}
```

Comment:
trivial/personal: As the method name is stub specific, I'd have named this key "resources".